db/migrate/20201224004047_create_top_contents.rb

```diff
@@ -0,0 +1,15 @@
+class CreateTopContents < ActiveRecord::Migration[6.0]
+  def change
+    create_table :top_contents do |t|
+      t.string :title
+      t.string :medium_type
```

Comment:
今更ながらcontent_typeのほうがわかりやすいかもしれません
```suggestion
      t.string :content_type
```